docs/advanced_topics/api/v2/usage.md

```diff
@@ -337,6 +337,7 @@ to only include descendants of that page (children, grandchildren etc.).
 ### Filtering pages by site
 
 ```{versionadded} 4.0
+
```

Comment:
this bit feels a tad odd, but not a big issue

Reply:
Yeah. I tested both outputs - with or without the spacing and it makes no visible difference. So I just went with what prettier did.